modelservice/src/main/sql/build-db.sql

```diff
@@ -58,4 +58,18 @@ create table user (
   password     varchar(100) not null,
   role         varchar(100),
   enabled      boolean
-);
\ No newline at end of file
+);
+
+create table metric_tags (
+tag_id        int NOT NULL AUTO_INCREMENT PRIMARY KEY , 
+tag_keys      varchar(255) NOT NULL, 
+tag_values    varchar(255) NOT NULL,
+unique index (tag_keys, tag_values)
+);
+
+create table metric_mapper(
```

Comment:
This table should have created_date column for audit purpose

Reply:
May be we need created_by as well - but would like @karanshah03 @williewheeler to confirm if this would be required and what values would that populate